app.php

```diff
@@ -0,0 +1,236 @@
+<?php
+require_once './vendor/autoload.php';
+use Appwrite\Client;
+use Appwrite\Services\Database;
+use Appwrite\Services\Storage;
+use Appwrite\Services\Users;
+
+$ch = curl_init();
+$certificate_location = "/usr/local/openssl-0.9.8/certs/cacert.pem";
```

Comment:
Are these required? Normally, the container is set up with proper certificates support. Furthermore, this would fail in many cases, because the cacert.pem isn't always installed at this location. I would just remove these 3 lines. It's not your job to set-up SSL support for cURL ;)